textual_fspicker/base_dialog.py

```diff
@@ -81,6 +85,14 @@ def __init__(
             select_button: Label for the select button.
         """
         super().__init__()
+
+        if os.name == 'nt':
+            import win32api
+            drivesStr = win32api.GetLogicalDriveStrings()
```

Comment:
I've never really written anything Windows-specific in Python; what is the likely "cost" in terms of time of `GetLogicalDriveStrings` returning? Is it something that could be very slow in some networked situations?

Reply:
GetLogicalDriveStrings

This function returns a concatenation of the drives in the Global and Local MS-DOS Device namespaces. If a drive exists in both namespaces, this function will return the entry in the Local MS-DOS Device namespace.

at the time level reads only the units recorded on the machine.
network drives are shown but you have to do a connection check

solution 1:
do the check if the drive is network using the GetDriveType function remove them from the list

solution 2:
do the check if the drive is network using the GetDriveType function 
only when you go to select the path and return an error notify

I just now realized that of this case.


### reference to the documentaiozne
[pywin32-docs/win32api__GetLogicalDriveString](https://timgolden.me.uk/pywin32-docs/win32api__GetLogicalDriveStrings_meth.html)

[win32/api/fileapi/nf-fileapi-getlogicaldrivestringsw](https://learn.microsoft.com/en-us/windows/win32/api/fileapi/nf-fileapi-getlogicaldrivestringsw)